pkg/controller/utils/elasticsearch.go

```diff
@@ -588,7 +589,11 @@ func getClientCredentials(client client.Client, ctx context.Context) (string, st
 	password := string(esSecret.Data[username])
 
 	// Determine the CA to use for validating the Elasticsearch server certificate.
-	roots, err := getESRoots(ctx, client, render.TigeraElasticsearchInternalCertSecret)
+	secretName := render.TigeraElasticsearchInternalCertSecret
+	if externalElastic {
+		secretName = render.TigeraElasticsearchInternalCertSecret
```

Comment:
Should this be TigeraElasticsearchExternalCertSecret?

Reply:
Yes, it should be.